packages/design-tokens/src/scss/themes/theme-light/_colors.scss

```diff
@@ -0,0 +1,421 @@
+$neutral-content-subtle: #f2f2f2 !default;
+$neutral-default: #0e2025 !default;
+$neutral-hover: #223439 !default;
+
+$selected-active: #7e4fe3 !default;
+$selected-content-basic: #5314da !default;
+$selected-content-subtle: #fff !default;
+$selected-default: #5314da !default;
+$selected-hover: #7543e1 !default;
+
+$shadows-shadow-100-color-01: #00000026 !default;
+$shadows-shadow-200-color-01: #0003 !default;
+$shadows-shadow-300-color-01: #00000040 !default;
+$shadows-shadow-400-color-01: #00000040 !default;
+
+$text-primary: #132930 !default;
+$text-secondary: #5c747b !default;
+$text-tertiary: #e4e4e4 !default;
+
+$action-colors: (
+    button: (
+        primary: (
+            active: $action-button-primary-active,
+            border: $action-button-primary-border,
+            content: $action-button-primary-content,
+            default: $action-button-primary-default,
+            hover: $action-button-primary-hover,
+        ),
+        secondary: (
+            active: $action-button-secondary-active,
+            border: $action-button-secondary-border,
+            content: $action-button-secondary-content,
+            default: $action-button-secondary-default,
+            hover: $action-button-secondary-hover,
+        ),
+        tertiary: (
+            active: $action-button-tertiary-active,
+            border: $action-button-tertiary-border,
+            content: $action-button-tertiary-content,
+            default: $action-button-tertiary-default,
+            hover: $action-button-tertiary-hover,
+        ),
+        plain: (
+            active: $action-button-plain-active,
+            border: $action-button-plain-border,
+            content: $action-button-plain-content,
+            default: $action-button-plain-default,
+            hover: $action-button-plain-hover,
+        ),
+    ),
+    link: (
+        primary: (
+            active: $action-link-primary-active,
+            default: $action-link-primary-default,
+            hover: $action-link-primary-hover,
+        ),
+        secondary: (
+            active: $action-link-secondary-active,
+            default: $action-link-secondary-default,
+            hover: $action-link-secondary-hover,
+        ),
+        tertiary: (
+            active: $action-link-tertiary-active,
+            default: $action-link-tertiary-default,
+            hover: $action-link-tertiary-hover,
+        ),
+        visited: (
+            default: $action-link-visited-default,
+        ),
+    ),
+    toggle: (
+        border-basic: $action-toggle-border-basic,
+        border-subtle: $action-toggle-border-subtle,
+        content-basic: $action-toggle-content-basic,
+        content-subtle: $action-toggle-content-subtle,
+        default-basic: $action-toggle-default-basic,
+        default-subtle: $action-toggle-default-subtle,
+        hover-basic: $action-toggle-hover-basic,
+        hover-subtle: $action-toggle-hover-subtle,
+    ),
```

Comment:
1. No `active` colors for toggle?
2. Usually we have a single `hover` (and `active`) color in similar sets instead of separate colors for each basic and subtle variants. But maybe it's needed here because there are two default colors.

Reply:
Good point, I opened a discussion with our designers and we can solve this one in another PR.